packages/api-client/src/constants.ts

```diff
@@ -0,0 +1,19 @@
+export const DATA_TIME_FORMAT = 'YYYY-MM-DD HH:mm:ss';
+
+export type EndpointBaseUrlSet = {
+  entity: string;
+  meditrak: string;
+  report: string;
+}
+
+export const LOCALHOST_ENDPOINT_BASE_URLS: EndpointBaseUrlSet = {
```

Comment:
For convenience, allows you to use the default set of localhost urls.

However most people will use this package via server-boilerplate, which reads and passes in the .env vars, and feature branches and production will do the same.